### PR TITLE
MAUI-2436 Write the UG documentation for conditional styling feature

### PR DESCRIPTION
**Pull request by FarjanaParveen**
MAUI-2436 Write the UG documentation for conditional styling feature

### CODE REVIEW

File: MAUI/DataGrid/conditional-styling.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+---
+
+# Conditional Styling in .NET MAUI DataGrid (SfDataGrid)
+The [SfDataGrid] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.html) allows to customize the style of the individual cells and rows based on the requirements. It can be customized in the following ways:
+
+1. Using column CellStyle
+2. Using Converter
+3. Using DataTriggers
+
+## Conditional Row Style
```
Comment (NeelakandanKannan) [inline MAUI/DataGrid/conditional-styling.md:17]: @FarjanaParveen  - Conditional row style

File: MAUI/DataGrid/conditional-styling.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+The record rows ([DataGridRow] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridRow.html)) can be customized conditionally by changing its property value based on ‘cell value’ or ‘data object’ by using converter, where converter returns the value based on underlying record.
```
Comment (NeelakandanKannan) [inline MAUI/DataGrid/conditional-styling.md:18]: @FarjanaParveen  - 

The data rows can be customized conditionally by writing the style with converter for [DataGridRow] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridRow.html) control. Its `BindingContext` is underlying datasource object.

File: MAUI/DataGrid/conditional-styling.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+    {
+        throw new NotImplementedException();
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Conditional DataGridRow styling in .NET MAUI DataGrid](Images/conditional-styling/maui-datagrid-conditional-datagridrowstyle.png)
+
+## How to style a particular row based on RowIndex
```
Comment (NeelakandanKannan) [inline MAUI/DataGrid/conditional-styling.md:54]: How to style a particular row based on row index

File: MAUI/DataGrid/conditional-styling.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+    {
+        throw new NotImplementedException();
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Conditional DataGridRow styling Based on RowData in .NET MAUI DataGrid](Images/conditional-styling/maui-datagrid-conditional-datagridrowstyle_basedonrowdata.png)
+
+## Conditional Style for Column
```
Comment (NeelakandanKannan) [inline MAUI/DataGrid/conditional-styling.md:126]: @FarjanaParveen  - Conditional style for particular column

File: MAUI/DataGrid/conditional-styling.md [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Conditional DataGridRow styling Based on RowData in .NET MAUI DataGrid](Images/conditional-styling/maui-datagrid-conditional-datagridrowstyle_basedonrowdata.png)
+
+## Conditional Style for Column 
+
+The `SfDataGrid` provides the support for apply the conditional style for a column by using [CellStyle] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridColumn.html#Syncfusion_Maui_DataGrid_DataGridColumn_CellStyleProperty) property in [SfDataGrid.Column] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridColumn.html).
```
Comment (NeelakandanKannan) [inline MAUI/DataGrid/conditional-styling.md:128]: @FarjanaParveen  - 
The `SfDataGrid` provides the support to apply the conditional style for specific column by using [CellStyle] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridColumn.html#Syncfusion_Maui_DataGrid_DataGridColumn_CellStyleProperty) property in [DataGridColumn] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridColumn.html).

**Review by NeelakandanKannan — CHANGES_REQUESTED** [review]
Get back to me with the changes.

**Review by NeelakandanKannan — APPROVED** [review]
The changes are fine.

File: MAUI/DataGrid/conditional-styling.md [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+layout: post
+title: Conditional Styling in .NET MAUI DataGrid control | Syncfusion
+description: Learn all about Conditional Styling support in Syncfusion .NET MAUI DataGrid (SfDataGrid) control, its elements and more.
+platform: MAUI
+control: SfDataGrid
+documentation: ug
+---
+
+# Conditional Styling in .NET MAUI DataGrid (SfDataGrid)
+The [SfDataGrid] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.html) allows to customize the style of the individual cells and rows based on the requirements.
```
Comment (Chrispineagunja) [inline MAUI/DataGrid/conditional-styling.md:11]: The [SfDataGrid] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.html) allows you to customize the style of the individual cells and rows based on the requirements.

File: MAUI/DataGrid/conditional-styling.md [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+platform: MAUI
+control: SfDataGrid
+documentation: ug
+---
+
+# Conditional Styling in .NET MAUI DataGrid (SfDataGrid)
+The [SfDataGrid] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.html) allows to customize the style of the individual cells and rows based on the requirements. 
+
+## Conditional row style
+The data rows can be customized conditionally by writing the style with converter for [DataGridRow] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridRow.html) control. Its BindingContext is underlying datasource object.
```
Comment (Chrispineagunja) [inline MAUI/DataGrid/conditional-styling.md:14]: The data rows can be customized conditionally by writing the style with a converter for the [DataGridRow] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridRow.html) control. Its BindingContext is the underlying datasource object.

File: MAUI/DataGrid/conditional-styling.md [hunk 8]
```diff
@@ -0,0 +1,168 @@
...
+        throw new NotImplementedException();
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Conditional DataGridRow styling in .NET MAUI DataGrid](Images/conditional-styling/maui-datagrid-conditional-datagridrowstyle.png)
+
+## How to style a particular row based on row index
+Styling can be applied to a particular row based on RowIndex property by writing the style for [DataGridRow] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridRow.html) TargetType.
```
Comment (Chrispineagunja) [inline MAUI/DataGrid/conditional-styling.md:51]: Styling can be applied to a particular row based on the RowIndex property by writing the style for the [DataGridRow] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridRow.html) TargetType.

File: MAUI/DataGrid/conditional-styling.md [hunk 9]
```diff
@@ -0,0 +1,168 @@
...
+        throw new NotImplementedException();
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Conditional DataGridRow styling Based on RowIndex in .NET MAUI DataGrid](Images/conditional-styling/maui-datagrid-conditional-datagridrowstyle_basedonrowindex.png)
+
+## How to style a particular row based on RowData
+Styling can be applied to a particular row based on RowData property by writing the style for [DataGridRow] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridRow.html) TargetType.
```
Comment (Chrispineagunja) [inline MAUI/DataGrid/conditional-styling.md:87]: Styling can be applied to a particular row based on the RowData property by writing the style for the [DataGridRow] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridRow.html) TargetType.

File: MAUI/DataGrid/conditional-styling.md [hunk 10]
```diff
@@ -0,0 +1,168 @@
...
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Conditional DataGridRow styling Based on RowData in .NET MAUI DataGrid](Images/conditional-styling/maui-datagrid-conditional-datagridrowstyle_basedonrowdata.png)
+
+## Conditional style for particular column
+
+The `SfDataGrid` provides the support to apply the conditional style for specific column by using [CellStyle] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridColumn.html#Syncfusion_Maui_DataGrid_DataGridColumn_CellStyleProperty) property in [DataGridColumn] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridColumn.html).
```
Comment (Chrispineagunja) [inline MAUI/DataGrid/conditional-styling.md:124]: The `SfDataGrid` provides the support to apply the conditional style for specific column by using the [CellStyle] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridColumn.html#Syncfusion_Maui_DataGrid_DataGridColumn_CellStyleProperty) property in the [DataGridColumn] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridColumn.html).

**Review by NeelakandanKannan — APPROVED** [review]
The changes are fine.

**Pull request merged**